projects/Mallard/src/screens/settings/join-beta-button.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Heading } from 'src/components/layout/ui/row'
+import { List } from 'src/components/lists/list'
+import { UiBodyCopy } from 'src/components/styled-text'
+import { Linking, StyleSheet } from 'react-native'
+import { JOIN_BETA_LINK } from 'src/constants'
+import { isInBeta } from 'src/helpers/release-stream'
+import remoteConfig from '@react-native-firebase/remote-config'
+import { metrics } from 'src/theme/spacing'
+
+const betaButtonStyle = StyleSheet.create({
+    thanksText: {
+        marginTop: metrics.vertical,
+        marginLeft: metrics.horizontal,
+    },
+})
+
+const betaThanks = () => (
+    <>
+        <UiBodyCopy style={betaButtonStyle.thanksText}>
+            Thank you for being a beta tester 🙌
+        </UiBodyCopy>
+        <Heading>{``}</Heading>
+    </>
+)
+
+const joinBetaMenuButton = () => (
+    <>
+        <Heading>{``}</Heading>
+        <List
+            data={[
+                {
+                    key: 'Become a beta tester 🙌',
+                    title: 'Become a beta tester 🙌',
+                    onPress: () => {
+                        Linking.openURL(JOIN_BETA_LINK) //what to catch here?
```

Comment:
As long it is running in a real device there should not be any exception, so I think we a fine without a `catch`